src/types.js

```diff
@@ -85,10 +100,18 @@ export type CIConfig = {
   REPO_SLUG: string
 };
 
+export type Settings = {
+  requireApproval: boolean,
+  requireClosedTasks: boolean,
+  requireGreenBuild: boolean,
+  usersAllowedToApprove: Array<string>
```

Comment:
I'll likely move the `usersAllowedToApprove` here at some point. This was just getting ahead of myself

Reply:
Note: this is done now